spec/chewy/index/actions_spec.rb

```diff
@@ -429,6 +429,12 @@
             expect(CitiesIndex).to receive(:import).with(suffix: suffix, journal: false, refresh: false).and_call_original
             expect(CitiesIndex.reset!(suffix)).to eq(true)
           end
+
+          specify 'uses empty index settings if not defined' do
+            allow(Chewy).to receive(:wait_for_status).and_return(nil)
+            allow(CitiesIndex).to receive(:settings_hash).and_return({})
```

Comment:
Did you check if it fails without your fix?

Reply:
Yeah, the root cause was because of spec/spec_helper.rb:27 ; if the :index option isn't set, then the tests fail because Index#index_settings tries to access it.